### PR TITLE
Initial IPR.2

### PR DESCRIPTION
**Pull request by TaranFurnell**

### CODE REVIEW

**Comment by TaranFurnell** [comment]
Can someone please review?

**Comment by jni2000** [comment]
We will review this.

File: src/harness/testcases/WINNF_FT_S_IPR_testcase.py [hunk 1]
```diff
@@ -236,9 +237,219 @@ def test_WINNF_FT_S_IPR_1(self, config_filename):
...
+
+    # Load Devices
+    device_a = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_a.json')))
+    device_a['installationParam']['latitude'] = 30.71570
+    device_a['installationParam']['longitude'] = -88.09350
+    device_b = json.load(
+        open(os.path.join('testcases', 'testdata', 'device_b.json')))
+    device_b['installationParam']['latitude'] = 30.71570
+    device_b['installationParam']['longitude'] = -88.09350
```
[the remark this answers is not in the document]
Comment (TaranFurnell) [reply src/harness/testcases/WINNF_FT_S_IPR_testcase.py]: Moved one of the CBSDs to be in a slightly different place.

**Comment by TaranFurnell** [comment]
I've addressed your comment, also updated the testcase to use the final piece of dpa integration (Still in review, PR #551).

**Comment by jni2000** [comment]
Taran. I am still in process of reviewing the PR. Will try to get back to you today or tomorrow.

File: src/harness/testcases/WINNF_FT_S_IPR_testcase.py [hunk 2]
```diff
@@ -251,8 +251,8 @@ def generate_IPR_2_default_config(self, filename):
     device_a['installationParam']['longitude'] = -88.09350
     device_b = json.load(
         open(os.path.join('testcases', 'testdata', 'device_b.json')))
-    device_b['installationParam']['latitude'] = 30.71570
-    device_b['installationParam']['longitude'] = -88.09350
+    device_b['installationParam']['latitude'] = 30.71571
+    device_b['installationParam']['longitude'] = -88.09351
```
Comment (jni2000) [inline src/harness/testcases/WINNF_FT_S_IPR_testcase.py]: Just a question: why are we making such small changes on CBSD location ?

Comment (TaranFurnell) [reply src/harness/testcases/WINNF_FT_S_IPR_testcase.py]: There's no minimum distance between CBSDs.

**Pull request merged**